### DISCUSSION TITLE
Betriebstellen/Wegpunkte

### DISCUSSION THREAD

**Discussion by embix** [marhei/TrainCompany-Data#413] [General]
Aktuell ist für Wegpunkte/Betriebsstellen kein "Art des Bahnhofs" int in  Station.json vorgesehen, obwohl diese einen Ril100 haben. Oftmals handelt es sich um ehemalige Bahnhöfe oder Anschlussstellen. Der Trassenfinder spuckt die auch mit aus und bei tcroute kommen die dann mit Leerstring.  Bisher hatte ich die entweder als Haltepunkt drin oder manuell wieder rausgenommen.

Als Haltepunkt zu behandeln ist doof, weil ein RB da normalerweise nicht hält - z.T. sind es auch ehemalige Haltepunkte (z.B. TZO Zollern) die ggf. wieder reaktivert werden. Im Falle einer Reaktivierung bräuchte man dann nur noch den Code zu ändern (und ggf. Bahnsteiglänge und -anzahl hinzufügen) und nicht nochmal die Strecke anfassen. Für die Strecke [Münster-Sendenhorst ist z.B. Personenverkehr-Reaktivierung im Gespräch](https://www.wn.de/muenster/wle-trasse-bekommt-haltepunkte-2586502).

Aktuell will ich gerade die Lücke zwischen Lübbenau und Calau stopfen (und später bis Bautzen runterziehen). Da liegt aber nur BBSD Bischdorf (Lausitz) zwischen. Jetzt hab ich die Wahl, ob ich den als Haltepunkt reinnehme, damit er die Strecke entsprechend auf er Karte genauer rendert (aber ein RB da halten würde) - oder weglasse und die beiden Bahnhöfe direkt miteinander verbinde. Anschlußstelle wäre hier vielleicht noch Option - paßt für einen deaktivierten Bahnhof aber auch nur bedingt und bei deaktivierten Haltepunkten gabs ja nicht mal ne Weiche. Mit nem extra Typ für Wegpunkte bräuchte ich jetzt nicht groß nachdenken und wäre bestimmt schon mit dem Streckenteilchen fertig 🙃.

TL;DR: Wollen wir einen extra Code für Wegpunkte einführen, die keine Haltepunkte sind einführen?

**Comment by marhei** [comment]
Du kannst das gerne mal als neuen Station-Gruppe 6 einfügen:
- Wird nicht auf der Karte gerendert
- Hat keine Bahnsteige

**Reply by embix** [reply]
Ist [das](https://github.com/marhei/TrainCompany-Data/blame/00459b728dc4af329ebdc2af7e017ed8ee70fc1a/README.md#L91) `§` anstelle der `6` dann ein Typo - oder hat sich das inzwischen geändert?

**Reply by marhei** [reply]
Das ist ein Typo.
